test/univariates.jl

```diff
@@ -81,18 +115,20 @@ function verify_and_test(d::UnivariateDistribution, dct::Dict, n_tsamples::Int)
         lp = Float64(pt["logpdf"])
         cf = Float64(pt["cdf"])
         Base.Test.test_approx_eq(logpdf(d, x), lp, "logpdf(d, $x)", "lp")
-        Base.Test.test_approx_eq(cdf(d, x), cf, "cdf(d, $x)", "cf")
+        if !isa(d, Skellam)
+            Base.Test.test_approx_eq(cdf(d, x), cf, "cdf(d, $x)", "cf")
+        end
     end
 
     try
         m = mgf(d,0.0)
-        @test m == 1.0
+        Base.Test.test_approx_eq(m, 1.0, "mgf(d, 0.0)", "1.0")
     catch e
         isa(e, MethodError) || throw(e)
     end
     try
         c = cf(d,0.0)
-        @test c == 1.0
+        Base.Test.test_approx_eq(c, 1.0, "cf(d, 0.0)", "1.0")
```

Comment:
Seems the problem was that `@test` failed for `Skellam` here, so it was switched to `test_approx_eq`. I reordered the definition expressions to be more numerically stable, so the original test should now pass.